roles/common/tasks/tfenv.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+- name: Check if tfenv is installed
+  stat:
+    path: "/home/{{ user }}/.tfenv/bin/terraform"
+  register: tfenv_existence
+
+- name: Download tfenv
+  git:
+    repo: 'https://github.com/tfutils/tfenv.git'
+    dest: "/home/{{ user }}/.tfenv"
+    version: master
+  register: tfenv_installation
+
+- name: Check tfenv installation
```

Comment:
You forgot to add the PATH to your profile here. 
You have to use the absolute path for tfenv. Not handy ;)

Reply:
I will take of this in next iteration (with zshrc templates and so on)